config/jail.conf

```diff
@@ -767,3 +767,16 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+port         = ftp,ftp-data,ftps,ftps-data
+# knocking URL needs to be customized in apache-pass.local
```

Comment:
I am confirming that I have pushed @szepeviktor to that frontier (now hard to find among all the comments ;) ).  It "needs to be" so users don't just enable it with default knocking string, which would open possibility for knowledgeable attackers to exploit it.